### PR TITLE
Allow save-images to optionally use podman

### PR DESCRIPTION
**Pull request by wgnathanael**
#### What this PR does / why we need it:
We use podman locally instead of docker, this change allows us to run the save-images.sh script using podman instead of docker without editing the script each time.

### CODE REVIEW

**Review by innobead — APPROVED** [review]
LGTM

**Comment by innobead** [comment]
@mantissahz Please help review this.

**Review by mantissahz — APPROVED** [review]
LGTM (Verified)
And @wgnathanael could we have a new version of `load-images.sh` as this one? That will be perfect.

Only need to modify the config file `/etc/containers/registries.conf` with `unqualified-search-registries = ["docker.io"]` for using the `podman`

**Comment by wgnathanael** [comment]
Ok, Added the load-images.sh as well.

File: scripts/save-images.sh [hunk 1]
```diff
@@ -1,5 +1,6 @@
 #!/bin/bash
 list="longhorn-images.txt"
+DOCKER_OR_PODMAN=${DOCKER_COMMAND:-docker}
```
Comment (derekbit) [inline scripts/save-images.sh:3]: nit: Rename `DOCKER_OR_PODMAN` to `CONTAINER_CLI`

Comment (wgnathanael) [reply scripts/save-images.sh]: applied in my latest force push

**Comment by innobead** [comment]
@c3y1huang can consider to add these scripts to our CLI.

**Comment by c3y1huang** [comment]
https://github.com/longhorn/longhorn/issues/7927

**Pull request merged**